content/introR-SWFLWMD-Sep16/placeholder.Rmd

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Yes because it's the only way to get the folder to show up. Once the `gsIntroR` pull requests are merged, I can actually add content.